packages/api/src/routes/events.ts

```diff
@@ -142,6 +152,19 @@ export function getTypeByEvent(): {[K in EventType]: Type<EventData[K]>} {
         epoch: "epoch",
       },
     }),
+
+    [EventType.lightclientUpdate]: new ContainerType<EventData[EventType.lightclientUpdate]>({
```

Comment:
How to obtain the initial latest update in case there is none for a while (missed blocks)? 
Or, is an initial update sent on subscription? And, are finalized / non-finalized updates pushed over the same stream?

Reply:
> How to obtain the initial latest update in case there is none for a while (missed blocks)?

The user can provide any checkpoint. if there's a long range of skipped slots just pick the latest checkpoint available. There's no need for this checkpoint to be in a particular time window, it could be 1 hour old or 1 day old. As long as the blockchain includes 1 single block there will exist at least one snapshot to initialize with.

> Or, is an initial update sent on subscription?

The current flow of the consumer is
1. Get init snapshot, get a sync committee
2. Get sync_committee updates from the period of the initial sync committee to current clock period
3. Subscribe to SSE lightclient_update feed to get header updates